lib/Installation/ProductSelection/AdditionalProductPage.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Additional Prodcuts page
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::ProductSelection::AdditionalProductPage;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my ($self) = @_;
+    $self->{btn_add} = $self->{app}->button({id => 'ok'});
+    return $self;
+}
+
+sub is_shown {
+    my ($self) = @_;
+    return $self->{btn_add}->exist();
```

Comment:
I suppose we can add more condition to confirm this page exist not only check ok button?

Reply:
I guess it's fine, this is how the `is_shown()` function is implemented in other pages as well.

Reply:
As this is new page, the background page don't have ok button. So use this can ensure the 'additional products' page pop up.

Reply:
BTW: Instead of writing it as a page, since it's a popup, you could also write it as a popup (essentially, it's just a page that just has "popup" in the file title - see the `lib/Installation/Popups` folder).

However it's not important, and you've done a great work here, so I'm not asking to change it - it looks good to me as it is.